src/lib/multicall/Multicall.ts

```diff
@@ -23,6 +22,11 @@ const CHAIN_BY_CHAIN_ID = {
   [AVALANCHE]: avalanche,
 };
 
+export type MulticallProviderUrls = {
+  default: string;
+  fallback: string;
```

Comment:
regarding naming
"default" also has "fallback" meaning, let's somehow distinguish it 

I'd suggest
"current" and "fallback"
"primary" and "secondary"
"main" and "fallback"

Reply:
another thought after looking at the rest of the code.
even after changing these names we still have `getBestRpc(...).fallback` which also doesn't make sense.

let's have `getBestRpc` (or `getBestRpcs`) internally how it is now, 
but let's also introduce `getRpcUrl` and `getFallbackRpcUrl` that would call `getBestRpc(s)` internally.

That way we don't need `MulticallProviderUrls` structure, at least outside of this file, as well as calling `getBestRpc` directly

EDIT `MulticallProviderUrls` is also needed in `src/lib/multicall/multicall.worker.ts`

Reply:
> I changed naming to the "primary" and "fallback" providers if it's ok with you

i think it's better to use "primary" and "secondary" then

Reply:
to be fair "default" and "fallback" also clear good to me

Reply:
renamed to "primary" and "secondary"